### ISSUE TITLE
Ability to specify the button order in the dialog

### ISSUE THREAD

**Issue by carstenhag** [afollestad/material-dialogs#1991]
**What module does this apply to?**

Core

**Description what you'd like to happen:**

In our apps we want to show a dialog which contains a title, message and two buttons.  
The message is similar to "Do you want to start Action A or Action B?".  
The buttons contain "Start Action A" and "Start Action B".  
As both texts are so long, in some device & language combinations they don't fit one line and get stacked.

The problem with them being stacked, is that the button order is now different. It shows like this:

|Do you want to start Action A or Action B|
|---:|
|Action B|
|Action A|

So to sum up, I'd love a property like "order" which makes sure a specific button is always shown either at the top or the left.

**Describe alternatives you've considered:**

The alternative would be to use the listItems thing. With the default list the buttons are left-aligned, which (without icon) is not Android-style. I could of course implement my own recyclerview and view for it, but the overhead of doing so, just to have the button orders fixed/specified, is huge.
